pkg/file/builder.go

```diff
@@ -1421,6 +1460,40 @@ func (b *stateBuilder) plugins() {
 			p.ConsumerGroup = utils.GetConsumerGroupReference(cg.ConsumerGroup)
 		}
 
+		if p.Partials != nil {
+			var pluginPartials []*kong.PartialLink
+			for _, partial := range p.Partials {
+				if partial.Partial != nil {
```

Comment:
To prevent deep nesting can we revert the condition here to `if partial.Partial == nil` and `continue` if that happens?